sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/properties/resourcemanager/AzureResourceMetadataConfigurationProperties.java

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.cloud.autoconfigure.properties.resourcemanager;
+
+import com.azure.spring.core.properties.resource.AzureResourceMetadata;
+
+/**
+ * Metadata defining an Azure resource.
+ */
+public class AzureResourceMetadataConfigurationProperties implements AzureResourceMetadata {
+
+    /**
+     * The resource group holds an Azure resource.
+     */
+    private String resourceGroup;
+    /**
+     * ID of an Azure resource.
+     */
+    private String resourceId;
+    /**
+     * The region of an Azure resource.
+     */
+    private String region;
+
+    public String getResourceGroup() {
```

Comment:
I guess we should add `@Override` annotation, otherwise it will be marked by API-review

Reply:
added